Source/EphysSocketEditor.cpp

```diff
@@ -31,63 +31,72 @@ EphysSocketEditor::EphysSocketEditor(GenericProcessor* parentNode, EphysSocket *
     portInput->setColour(Label::backgroundColourId, Colours::lightgrey);
     portInput->setEditable(true);
     portInput->addListener(this);
-    portInput->setBounds(10, 70, 65, 15);
+    portInput->setBounds(10, 73, 65, 15);
     addAndMakeVisible(portInput);
 
-    //---
+    //--- BUFFER SIZE
     bufferSizeMainLabel = new Label("BUFFER SIZE", "BUFFER SIZE");
     bufferSizeMainLabel->setFont(Font("Small Text", 12, Font::plain));
-    bufferSizeMainLabel->setBounds(114, 30, 95, 15);
+    bufferSizeMainLabel->setBounds(114, 25, 95, 15);
     bufferSizeMainLabel->setColour(Label::textColourId, Colours::darkgrey);
     addAndMakeVisible(bufferSizeMainLabel);
 
     // Num chans
-    channelCountLabel = new Label("CHANNELS", "CHANNELS");
+    channelCountLabel = new Label("CHANNELS", "Channels");
```

Comment:
Why make this type of aesthetic change? I think the current all caps was probably just keeping in line with other looks and feel conventions in the GUI

Reply:
Now that i built and am looking at it, I understand. Looks good in the context where its needed. However, in the latest revision all this data is in the header and these options should be removed from the GUI